packages/sdk/src/actor/physics/collider.ts

```diff
@@ -46,6 +47,9 @@ export enum CollisionLayer {
 export interface ColliderLike {
 	enabled: boolean;
 	isTrigger: boolean;
+	bounciness: float;
```

Comment:
Instead of importing `float`, just make these type `number`.

Reply:
done, I use numbers